stable/postgresql/templates/statefulset.yaml

```diff
@@ -8,10 +8,29 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
+  serviceName: {{ template "postgresql.fullname" . }}
+  selector:
+    matchLabels:
+      app: {{ template "postgresql.fullname" . }}
   template:
     metadata:
       labels:
         app: {{ template "postgresql.fullname" . }}
+    volumeClaimTemplates:
```

Comment:
Maintains use of the `values.yaml` file for configuring aspects of the `volumeClaimTemplate`